tests/Unit/Visualization/Python/CMakeLists.txt

```diff
@@ -5,3 +5,8 @@ spectre_add_python_bindings_test(
   "Unit.Visualization.Python.GenerateXdmf"
   Test_GenerateXdmf.py
   "unit;visualization;python")
+
+spectre_add_python_bindings_test(
```

Comment:
I don't really know how the python tests work, but this isn't a bindings test, is it?